### PR TITLE
added first draft of donation post feed and function to get donation …

### PR DESCRIPTION
**Pull request by mertdayi01**
…post data

### CODE REVIEW

**Review by ysheliakin — CHANGES_REQUESTED** [review]
I think this a good start with some minor things to correct

File: views/pages/index.html [hunk 1]
```diff
@@ -15,6 +15,7 @@
     <main class="container" id="page">
         {{template "hero" .}}
         {{template "about" .}}
+        {{template "donationPostFeed" .}}
```
[the remark this answers is not in the document]
Comment (mertdayi01) [reply views/pages/index.html:18]: ok i removed it.

File: main.go [hunk 2]
```diff
@@ -66,6 +66,12 @@ func main() {
 	e.GET("/", func(c echo.Context) error {
 		return c.Render(200, "index", nil)
 	})
+	e.GET("/donationPostFeed", func(c echo.Context) error {
```
[the remark this answers is not in the document]
Comment (mertdayi01) [reply main.go:69]: sounds good. changed it to feed

File: main.go [hunk 3]
```diff
@@ -66,6 +66,12 @@ func main() {
...
+    return handlers.GetDonationPosts(dbc, query, c, e.Logger)
+	})
+	e.POST("/donationPostFeed", func(c echo.Context) error {
```
[the remark this answers is not in the document]
Comment (mertdayi01) [reply main.go:72]: i wasn't exactly sure what it was doing so i added it. ill try removing it.

File: db/queries/donation.sql [hunk 4]
```diff
@@ -6,3 +6,6 @@ values (
     $1, $2, NOW(), $3, $4, $5, $6, $7, $8, $9, $10
 )
 returning id;
+
+-- name: GetDonations :many
+SELECT * FROM donations;
```
[the remark this answers is not in the document]
Comment (mertdayi01) [reply db/queries/donation.sql:11]: i'm not sure what else to do.

File: views/components/donationPostFeed.html
[the remark this answers is not in the document]
Comment (mertdayi01) [reply views/components/donationPostFeed.html:1]: moved it

**Review by ysheliakin — APPROVED** [review]

**Pull request merged**